app/controllers/admin/organisations_controller.rb

```diff
@@ -22,14 +23,25 @@ def destroy
 private
 
   def sortable_columns
-    %w[name created_at active_storage_attachments.created_at]
+    %w[name created_at active_storage_attachments.created_at address]
   end
 
   def sort_column
-    sortable_columns.include?(params[:sort]) ? params[:sort] : "created_at"
+    sortable_columns.include?(params[:sort]) ? params[:sort] : sortable_columns[1]
+  end
+
+  def sort_column_show
+    sortable_columns.include?(params[:sort]) ? params[:sort] : sortable_columns.last
```

Comment:
I think all of these `sort_*` methods have been misnamed.
One set relates to MoUs, another set relates to `locations`.

They're also using a central definition of sortable columns.
What happens if I try to sort MoUs by address?
What happens if I try to sort locations by `active_storage_attachments.created_at`.

Can these be fully broken up and made clear what's going on please?